### DISCUSSION TITLE
Why doesn't `backout` set the working copy to the new commit?

### DISCUSSION THREAD

**Discussion by BatmanAoD** [jj-vcs/jj#2911] [Q&A] [answered]
I found this behavior surprising: given a commit sequence like this:

```
@ [empty]
│
◉ A
│
◉ B
...
```

Running `jj backout -r B` results in this sequence:

```
◉ backout of commit B
│
@ [empty]
│
◉ A
│
◉ B
...
```

From this point, I would of course want to abandon the current `@` and checkout the "backout" commit. Why isn't checking out the `backout` commit the default behavior? Is there a way to make abandoning the original empty `@` commit automatic?

**Comment by PhilipMetzger** [comment, marked as answer]
This is a bug, see the corresponding issue in #2802.
